### DISCUSSION TITLE
UI is zoomed out and very small elements

### DISCUSSION THREAD

**Discussion by lancerdoom1** [upscayl/upscayl#610] [Q&A]
Hi all, 
I accidentally pressed "CTRL" plus "-" to zoom out the picture but i results in very small UI screen and text is also very small. how to reset the UI

![image](https://github.com/upscayl/upscayl/assets/29885113/32b49d3c-7711-4fc8-a97a-ad7ad57e8144)

**Comment by NayamAmarshe** [comment]
Press Ctrl + 0
